package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "bartop",
+  "version": "0.0.1",
+  "description": "Web framework for hobbyist and freelance bartenders",
+  "scripts": {
+    "install": "concurrently 'cd client && npm install' 'cd server && npm install'",
+    "start": "concurrently 'npm run server' 'npm run client'",
+    "server": "cd server && npm start",
+    "client": "cd client && npm start",
+    "test-server": "cd server && npm test",
+    "test-client": "cd client && CI=true npm test",
+    "test": "concurrently 'npm run test-server' 'npm run test-client'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dpopp07/bartop.git"
+  },
+  "keywords": [
+    "bartop"
+  ],
+  "author": "Dustin Popp",
```

Comment:
I know it's my birthday, but can I not be the author on all of these? Makes me feel like a clown

Reply:
lol I'll switch it to Hannah Banana